pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -58,16 +58,34 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	a.Log.Debug("Cluster roles completed")
 
 	for _, csv := range a.csvSet(op.Namespace, v1alpha1.CSVPhaseAny) {
-		origCSVannotations := csv.GetAnnotations()
+		origCSVannotations := a.copyOperatorGroupAnnotations(&csv.ObjectMeta)
 		a.addOperatorGroupAnnotations(&csv.ObjectMeta, op, !csv.IsCopied())
 		if reflect.DeepEqual(origCSVannotations, csv.GetAnnotations()) == false {
```

Comment:
Will this do the right thing if there are other annotations on the csv? (Non-operatorgrou-related)

Reply:
Ah right, I need to make a getter. The intention was to really only test operator group annotations.